### PR TITLE
update logo

### PR DESCRIPTION
**Pull request by merittrompeter**
Matching logo on DevCenter with logo on DevCenter Forums.

### CODE REVIEW

File: _includes/top-navigation.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
                         <span class="fa fa-bars"></span>
                     </button>
                     <a class="navbar-brand" href="{{ site.baseurl }}/">
-                        <img id="logo-header" src="{{ site.baseurl }}/concur_assets/img/Concur_Logo_VT_Color_40px.png" alt="Logo">
+                        <img id="logo-header" src="{{ site.baseurl }}/concur_assets/img/concur_logo.gif" alt="Logo">
```
Comment (howarddierking) [inline _includes/top-navigation.html]: does this work out to be the same dimensions as the current, or does it change the overall header dimensions?

File: assets/css/headers/header-default.css [hunk 2]
```diff
@@ -120,10 +120,30 @@
...
+	.header .navbar-brand {
+		white-space: pre-line;
+		top: -42px;
+		margin-bottom: -36px;
+	}
+}
+
+@media(min-width: 767px) and (max-width: 998px) {
+	.header .navbar-brand {
 		margin-top: 0;
```
Comment (howarddierking) [inline assets/css/headers/header-default.css]: This seems to be the other major substantive change in this PR - can you describe the change a bit more?

Comment (merittrompeter) [inline assets/css/headers/header-default.css]: Sure! These styling changes were meant to accomplish two things:
a) make the text in the logo and the text in the remainder of the header line up when inline
b) change the text in the header to display beneath the logo as appropriate when the window resizes

File: _includes/top-navigation.html (hunk 1 above)
Comment (merittrompeter) [inline _includes/top-navigation.html]: This image is indeed bigger, so the header dimensions did increase.

File: assets/css/headers/header-default.css (hunk 2 above)
Comment (howarddierking) [inline assets/css/headers/header-default.css]: cool - so you verified that this looks good on mobile screen dimensions?

File: _includes/top-navigation.html (hunk 1 above)
Comment (howarddierking) [inline _includes/top-navigation.html]: @henryyamamoto-concur, did you review and are you good with the size changes?

File: assets/css/headers/header-default.css (hunk 2 above)
Comment (merittrompeter) [inline assets/css/headers/header-default.css]: Absolutely!

**Comment by nicklathe** [comment]
Everything looks good for me. :shipit:!

**Comment by henryyamamoto-concur** [comment]
Who's merging?

**Pull request merged**